lib/generators/tailwindcss/controller/controller_generator.rb

```diff
@@ -3,7 +3,7 @@
 module Tailwindcss
   module Generators
     class ControllerGenerator < Erb::Generators::ControllerGenerator
-      source_root File.expand_path("../templates", __FILE__)
+      source_root File.expand_path("templates", __dir__)
```

Comment:
Should we do the same here as in scaffold, i think it is something like this
`source_paths << "lib/templates/erb/controller"`

Reply:
Fixed